app/views/posts/_post.html.slim

```diff
@@ -0,0 +1,27 @@
+- @ticket_id = "ticket-#{post.id}"
+- @id = random_dom_id
+.ticket_box class=@ticket_id id=@id
+	= link_to post, {remote: true} do
+		.container-fluid
+			.row
+				.col-md-3
+					.profile_container
+						img class='profilepic_small' src="#{post.user.image}?sz=100"
+						.ticket_name #{post.user.first_name}
+				.col-md-9 class='ticket_description'
+					div.ticket_icon
+						img src=image_path('speechbubble.png')
+						.num_comments #{post.comments.count}
+					div.ticket_icon.follow
+						- post_followed = post.follows.exists?(user_id:current_user)
+
+						img src=image_path('followcheck.png') style="display:#{:none unless post_followed};"
+					.description #{post.short}
+					.tags 
+						= post.tag_list.map{ |tag| "<span class='tag'>#{ tag }</span>" }.join(", ").html_safe
+// CoffeeScript doesn't handle string interpolation well, so we create variables in native javascript here
+javascript:
+  var ticket_container =  "##{@id}.#{@ticket_id}";
+coffee:
+	new Spaces.AjaxLinkController ticket_container, (data) -> 
+		($ '.right_panel_content').html data
```

Comment:
eyh? what's wrong with
```
coffee:
  new Spaces.AjaxLinkController "##{ @id }", (data) ->
...
```
btw `@id` is guaranteed to be unique in the DOM so you shouldn't need to select on the class as well.